service/controller/internal/templates/cloudformation/tccp/record_sets.go

```diff
@@ -67,6 +67,15 @@ const RecordSets = `
       Name: '{{ $v.EtcdDomain }}.'
       HostedZoneId: !Ref 'HostedZone'
       Type: A
+  EtcdNodeRecordSet:
+    Type: AWS::Route53::RecordSet
+    Properties:
+      ResourceRecords:
+      - !GetAtt {{ $v.MasterInstanceResourceName }}.PrivateIp
```

Comment:
This is only needed at creation, IIUC? Could it happen that this IP changes and then the path can't resolve?

Reply:
We don't run the master in an ASG in `legacy`. It's a statically defined instance in the CloudFormation stack so the only time the IP would change would be if the master is terminated. We have a complex ops-recipe for this scenario, might be worth adding updating the record IP to the recipe if this PR is accepted.